docs/swagger.yaml

```diff
@@ -105,88 +112,128 @@ paths:
+
       parameters:
-        - $ref: "#/components/parameters/JWT"
         - $ref: '#/components/parameters/Vipps-System-Name'
         - $ref: '#/components/parameters/Vipps-System-Version'
         - $ref: '#/components/parameters/Vipps-System-Plugin-Name'
         - $ref: '#/components/parameters/Vipps-System-Plugin-Version'
-        - in: query
+        - in: path
+          name: LedgerId
           required: true
-          name: ledgerId
           schema:
             type: string
-        - in: query
-          required: false
-          name: ledgerDate
+            description: Ledger ID provided by GET:/settlement/v1/ledgers
+        - in: path
+          name: LedgerDate
+          required: true
           schema:
             type: string
-            format: Date
-            example: 2022-01-20
-        - in: query
-          required: false
-          name: fromLedgerDate
+            pattern: '^\d{4}-\d{2}-\d{2}'
+            description: Date in format yyyy-MM-dd
+        - in: path
+          name: Account
+          required: true
           schema:
             type: string
-            format: Date
-            example: 2022-01-19
+            enum: ["funds", "fees"]
+            description: One of `funds`, `fees`
         - in: query
           required: false
-          name: toLedgerDate
+          name: cursor
           schema:
             type: string
-            format: Date
-            example: 2022-01-22
-        - in: query
-          required: false
-          name: inPayout
+            example: "eyJhZnRlckxlZGdlcklkIjoieHh4eCIsInNldHRsZXNGb3JSZWNpcGllbnRIYW5kbGVzIjpudWxsfQ=="
+            description: |-
+              We limit the amount of rows returned to reduce the amount of data we need to process on each request. 
+              If the total number of rows is greater 'than that amount we return a cursor in addition to
+              the rows in the body. This cursor given can be used with this parameter to fetch the next set of ledgers.
+              If this is used, no other parameters is needed.
+      responses:
+        200:
+          $ref: "#/components/responses/200"
+        404:
+          description: The report for this date is not available yet
+        401:
+          $ref: "#/components/responses/401"
+        500:
+          $ref: "#/components/responses/500"
+        default:
+          description: Default problem response
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+
+  "/report/v2/ledgers/{LedgerId}/{Account}/feed":
+    get:
+      tags:
+        - /report/v2/ledgers
+      summary: Fetch report by feed
+      description: |-
+        Continuous stream of data. Upon reaching the end of the feed the `cursor` will not become empty and `items` can 
+        become en empty list. One should retry to fetch the same `cursor` until new data and a new `cursor` will be returned.
```

Comment:
Might just be a spelling error but I am not sure what is meant here. "en empty list"